packages/react-components/index.js

```diff
@@ -68,6 +68,7 @@ module.exports = (neutrino, opts = {}) => {
           .hints('error')
           .end()
         .output
+          .path(neutrino.options.output)
```

Comment:
Isn't this caught by the web preset?

Reply:
https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/web/index.js#L134

Reply:
Yes, that was a silly PR. Closing this now :)

Reply:
lol no worries!